ADR/0035-artifact-streams.md

```diff
@@ -0,0 +1,143 @@
+# 32. Decoupling Deployment
+
+Date started: 2023-11-17
+Date accepted: 2023-02-01
```

Comment:
Is this supposed to be 2024 or is the accepted date actually supposed to be after the started date?

Reply:
Yes, thanks. I had forgotten that I copied the previous ADR for structure.